xcashu/client/client.py

```diff
@@ -36,14 +36,15 @@ async def request_wrapped(wallet, *args, **kwargs):
 
     s.headers.update({"X-Cashu": token})
     resp = s.request(*args, **kwargs)
-
-    await wallet.invalidate(proofs, check_spendable=False)
```

Comment:
I think this is a bug? It did invalidate proofs even if the response indicated an error